string mode
string class_name
---
bool return